pkg/stanza/fileconsumer/internal/util/util.go

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package util // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/fileconsumer/internal/util"
```

Comment:
"Util" is a bit vague. Perhaps "compression" would be better (assuming if we ever support other than "gzip" compression methods, this would be their place too)?